ChangeLog.d/fix-test-suite-pk-warnings.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix redefinition warnings when SECP192R1 and/or SECP192K1 are disabled.
+     Fixes #9029.
```

Comment:
The last line is missing a newline at the end. That confuses some tools including `assemble_changelog.py`.